cli/src/cli.rs

```diff
@@ -82,6 +85,30 @@ where
                         println!("[error] {}\n", e);
                     }
                 },
+                Command::ExecuteFromFile(filename) => {
+                    if let Err(e) = self.load(&filename) {
+                        println!("[error] {}\n", e);
+                    }
+                }
+            }
+        }
+
+        Ok(())
+    }
+
+    pub fn load(&mut self, filename: &str) -> Result<()> {
+        let mut sqls = String::new();
+        // We open the file
+        File::open(filename)?.read_to_string(&mut sqls)?;
+        // We get all the sql query
```

Comment:
Your codes are good & obvious enough, so it works itself as a document. These two supplementary comments look unnecessary.